x/cdp/abci.go

```diff
@@ -20,6 +20,16 @@ func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k keeper.Keeper)
 
 	params := k.GetParams(ctx)
 
+	// only run CDP Begin Blocker every `BeginBlockerExecutionBlockInterval` blocks
+	blockHeight := ctx.BlockHeight()
+
+	if blockHeight%params.BeginBlockerExecutionBlockInterval != 0 {
+		ctx.Logger().Info("skipping x/cdp begin blocker")
+		return
+	}
+
+	ctx.Logger().Debug("running x/cdp begin blocker")
```

Comment:
I'm not sure how useful this is, but fine to leave.  But could be biased as I tend to lean towards no logs 😓